### PR TITLE
QoL Changes and fixes to the README

### PR DESCRIPTION
**Pull request by cryptidv**
Updating README to attempt to make neater.

### Background
The README needs some changes to structure and quality.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] ~~I have thoroughly tested my changes with multiple different prompts.~~
- [x] I have considered potential risks and mitigations for my changes.
- [x] ~~I have documented my changes clearly and comprehensively.~~
- [x] ~~I have not snuck in any "extra" small tweaks or changes~~

### CODE REVIEW

**Comment by cryptidv** [comment]
@richbeales sorry could you just merge this? This slightly updates PR #794 to make the points neater

File: README.md
Comment (lekapsy) [inline README.md:1]: Maybe update this as well?

## 📋 Requirements
- [Python 3.8 or later](https://www.tutorialspoint.com/how-to-install-python-in-windows)
- [OpenAI API key](https://platform.openai.com/account/api-keys)
- [PINECONE API key](https://www.pinecone.io/)

Comment (sadmuphin) [reply README.md:1]: Yes I'll add this too, good idea. 👍

**Review by nponeccop — APPROVED** [review]

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
@cryptidv There are conflicts again.

**Comment by cryptidv** [comment]
Closing. Way too outdated 😅

**Pull request closed without merging**